### PR TITLE
[stable/velero] update to v1.0.0

### PR DESCRIPTION
**Pull request by sebastien-prudhomme**
#### What this PR does / why we need it:

Update Velero to v1.0.0

#### Special notes for your reviewer:

The Velero documentation web site is not yet updated with v1.0.0 but the links should be working when it will be done.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] title of the PR contains starts with chart name e.g. `[stable/chart]`

### CODE REVIEW

File: stable/velero/README.md [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
 
-This helm chart installs Velero version v0.11.0
-https://github.com/heptio/velero/tree/v0.11.0
+This helm chart installs Velero version v1.0.0
+https://github.com/heptio/velero/tree/v1.0.0
 
 
+## Upgrading to v1.0.0
+
+As of v1.0.0, Heptio Velero is no more backwards-compatible with Heptio Ark.
```
Comment (nrb) [inline stable/velero/README.md]: s/no more/no longer/

File: stable/velero/README.md [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
+
+The [instructions found here](https://heptio.github.io/velero/v1.0.0/upgrade-to-1.0) will assist you in upgrading from version v0.11.0 to v1.0.0
```
Comment (nrb) [inline stable/velero/README.md]: We need to do a more detailed sweep of these docs, but for now the 1.0 docs live at https://velero.io/docs/v1.0.0/upgrade-to-1.0/

**Review by nrb — COMMENTED** [review]
A couple small changes, but otherwise I'm fine with this.

File: stable/velero/README.md (hunk 1 above)
Comment (sebastien-prudhomme) [reply stable/velero/README.md]: Done

File: stable/velero/README.md (hunk 2 above)
Comment (sebastien-prudhomme) [reply stable/velero/README.md]: I've corrected all the links

**Pull request merged**